src/Behat/Behat/Transformation/Transformation/ColumnBasedTableTransformation.php

```diff
@@ -32,17 +32,10 @@ public static function supportsPatternAndMethod($pattern, ReflectionMethod $meth
         return 1 === preg_match(self::PATTERN_REGEX, $pattern);
     }
 
-    /**
-     * Initializes transformation.
-     *
-     * @param string      $pattern
-     * @param callable    $callable
-     * @param string|null $description
-     */
     public function __construct(
-        private $pattern,
-        $callable,
-        $description = null,
+        private readonly string $pattern,
+        callable|array $callable,
```

Comment:
why `callable|array` while the phpdoc said `callable` ?

Reply:
`callable` accepts any callables, not just Closure.

If tests fail, it might indicate that we have places passing an array that is not actually callable.